Sources/Public/Deprecations.swift

```diff
@@ -0,0 +1,302 @@
+// Created by Bryan Keller on 12/22/21.
```

Comment:
Added this file to keep nearly all of the deprecations in one place. v2.0.0 will simply delete this file.